examples/nidaqmx/analog-input.py

```diff
@@ -76,7 +84,8 @@
         )
     )
 
-    client.StartTask(nidaqmx_types.StartTaskRequest(task=task))
+    start_task_response = client.StartTask(nidaqmx_types.StartTaskRequest(task=task))
```

Comment:
As I mentioned before, `StopTask` may return a specific warning. It's way at the bottom of each file in the `finally` block, but it doesn't really belong there because `ClearTask` is sufficient in error cases.

Reply:
I've removed the `StopTask` calls and am now checking the response of `ClearTask` for warnings.

Reply:
`ClearTask` does not report warnings. Calling `StopTask` in success cases gives the task a chance to report warnings.

When in doubt, follow the LV shipping examples.
![image](https://user-images.githubusercontent.com/17914399/189737025-901d656d-ef37-4aed-881f-86532311acb5.png)